draft-ietf-ppm-dap.md

```diff
@@ -615,6 +615,7 @@ standard tokens for use in the "type" field (within the DAP URN namespace
 | unauthorizedRequest        | Authentication of an HTTP request failed (see {{request-authentication}}). |
 | stepMismatch               | The Aggregators disagree on the current step of the DAP aggregation protocol. |
 | batchOverlap               | A request's query includes reports that were previously collected in a different batch. |
+| aggregationJobTooLarge     | An aggregation request contained too many report shares. |
```

Comment:
nit: not every aggregation request has report shares (only the first does).
1. If the intent is that we abort only on the initial request, then say "initial" here
2. (my preference): Use more agnostic language, e.g., "An aggregation job contains more reports than the negotiated threshold."